dlio_benchmark/checkpointing/tf_checkpointing.py

```diff
@@ -42,11 +58,11 @@ def __init__(self):
         super().__init__("pb")
 
     @dlp.log
-    def get_tensor(self, size):
-        return tf.random.uniform((int(size / 4),), maxval=100, dtype=tf.dtypes.int32)
+    def get_tensor(self, size, datatype="int8"):
```

Comment:
The API is different from what I had. What I mean is size is inclusive of type of data. Based on what you want to do, you can do array_length? or length instead of size.

Reply:
change size to length